Deutsches Arzteblatt.js

```diff
@@ -0,0 +1,255 @@
+{
+	"translatorID": "72543881-a05b-4a40-b891-de8726e29d77",
+	"label": "Deutsches Ärzteblatt",
+	"creator": "Sebastian Fleer",
+	"target": "^https?://www\\.aerzteblatt\\.de/archiv/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-11-14 09:17:33"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Sebastian Fleer
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.match(/archiv\/\d+/) || url.match(/aid\=\d+/)) {
+		return "journalArticle";
+	}
+}
+
+function doWeb(doc, url) {
+	var type = detectWeb(doc, url);
+	if (type == "journalArticle") {
```

Comment:
\The connector calls `detectWeb` and then `doWeb`, so this translator is running `detectWeb` three times: once in the connector, once in `doWeb`, and once in `scrape`. All calls but the first are redundant: `doWeb` will never be called if `detectWeb` doesn't return a type, and the only type it can return is `journalArticle`.